frontend/src/metabase/visualizations/visualizations/PieChart/DimensionsWidget.tsx

```diff
@@ -128,12 +129,16 @@ export function DimensionsWidget({
   const [draggedDimensionIndex, setDraggedDimensionIndex] = useState<number>();
 
   const onDragStart = (event: DragStartEvent) => {
+    document.body.classList.add(GrabberS.grabbing);
```

Comment:
This is the same approach as used in [`SortableList.tsx`](https://github.com/metabase/metabase/blob/40ed53ef8e9ae774cfbccb818d58f3f59565a232/frontend/src/metabase/core/components/Sortable/SortableList.tsx#L87)